src/panel/ServicePanel.jsx

```diff
@@ -8,11 +8,13 @@ class ServicePanel extends React.Component {
   render() {
     return <Panel
       resizable
-      title={_('Qwant Maps services', 'service panel')}
+      title=''
```

Comment:
I'm not convinced… For me routes, favorites, and now "good plans" are Qwant Maps "services" as well.
And for the scrolling thing, I agree it doesn't look right but could be made more pleasant, for example with a drop shadow. BTW it's how the category list is now and it looks ok.

I also think it's a good thing from a UX point of view that panels have a title which stays visible at all time, so users understand in which context they are. And it would be nice to reach for more cohesion, instead of introducing specific look and feel for each individual panel.